### ISSUE TITLE
Things To Do

### ISSUE THREAD

**Issue by elswork** [ml-hispano/MLH_bot#8]
- [x] Comando nuevo  /leaderboard_thisweek
- [x] Comando nuevo  /leaderboard_alltime
- [ ] Comando nuevo /leaderboard_fullranking

**Comment by pochocosta** [comment]
Están seguros de hacer un Full Ranking teniendo mas de 2000 personas en la comunidad? Me parece demasiado ja!

**Comment by elswork** [comment]
Cierto, la idea era ampliar las posiciones del ránking para que más gente pudiera ver su puesto, quizás un comando para que cada uno pueda consultarle al bot cual es su posición sería lo ideal.

**Comment by dotcsv** [comment]
Se puede hacer un /leaderboard_myposition que muestre la tabla del ranking con +/- 5 posiciones desde la posición del usuario que lanza el comando.

**Comment by andfk** [comment]
yo propongo `/leaderboard_me` (para que sea más fácil recordar)
